src/mono/System.Private.CoreLib/src/Mono/HotReload.cs

```diff
@@ -100,14 +100,14 @@ private FieldStore (object? loc)
 
     public static FieldStore Create (RuntimeTypeHandle type)
     {
-        Type t = Type.GetTypeFromHandle(type) ?? throw new ArgumentException(nameof(type), "Type handle was null");
+        Type t = Type.GetTypeFromHandle(type) ?? throw new ArgumentException(SR.Format(SR.Arg_NullTypeHandleReferenceException, nameof(type)));
```

Comment:
```suggestion
        Type t = Type.GetTypeFromHandle(type) ?? throw new ArgumentException(SR.Arg_InvalidHandle, nameof(type)));
```
Reuse existing string used in similar situations.